addons/mail/tests/common.py

```diff
@@ -120,3 +120,34 @@ def tearDownClass(cls):
         cls.env['ir.mail_server']._revert_method('build_email')
         cls.env['ir.mail_server']._revert_method('send_email')
         super(TestMail, cls).tearDownClass()
+
+    def gateway_reply_wrecord(self, template, record, extra_irt=True):
+        """ Simulate a reply through the mail gateway. Usage: giving a record,
+        find an email sent to him and use its message-ID to simulate a reply. """
+        email = self._find_sent_email_wrecord(record)
+
+        if extra_irt:
+            extra = 'In-Reply-To:\r\n\t%s\n' % email['message_id']
+        else:
+            disturbing_other_msg_id = '<123456.654321@another.host.com>'
```

Comment:
What is the purpose of the `disturbing_other_msg_id` ?
Verifying adding stuff doesn't break everything?

Reply:
Indeed. Docstring updated.